bin/reth/src/node/mod.rs

```diff
@@ -241,6 +241,8 @@ impl Command {
                     client,
                     pool,
                     chain_events,
+                    ctx.task_executor.clone(),
+                    Default::default(),
                 ),
```

Comment:
dont love the way we instantiate the txpool rn - would strongly pref builder pattern in the future